### PR TITLE
Iss796: Remove all author tags from source files

### PR DESCRIPTION
**Pull request by JeremyMcCormick**
See #796 for details.

Also, cleaned up class comments a bit in effected source files.

To see all users who have committed changes to a particular file use: `git log path/to/file`

### CODE REVIEW

**Review by pbutti — APPROVED** [review]

**Comment by pbutti** [comment]
Hey Jeremy, out of curiosity: why did we remove the author-tags? 
I'll take care of not adding them to the next pushes! 

Cheers,
PF

**Comment by JeremyMcCormick** [comment]
I listed the reasons in the linked issue.

**Comment by JeremyMcCormick** [comment]
Main issue is that inclusion of email addresses in these tags causes the javadoc to generate an enormous number of errors. Also, everyone who has ever changed a file is listed in the git log, so that's a better way to find out who to blame. It is also extra boilerplate which just isn't needed and somewhat implies that HPS code is owned by individuals, which it isn't. Those are most of the reasons I can think of right now. :)

**Comment by JeremyMcCormick** [comment]
The 'git log' command should tell you who created any file in the repo, and it should work even for old files that haven't been updated since the SVN days, since I mapped all the SVN users to github accounts when we imported.

**Comment by JeremyMcCormick** [comment]
Watch about loading the file changes tab in your browser. It may cause it to freeze. :-p

File: analysis/src/main/java/org/hps/analysis/MC/FirstHitGBLRefitterDriver.java [hunk 1]
```diff
@@ -254,4 +250,4 @@ static RelationalTable getHitToRotatedTable(EventHeader event, String RotatedHel
         }
         return hits;
     }
-}
\ No newline at end of file
+}
```
Comment (cbravo135) [inline analysis/src/main/java/org/hps/analysis/MC/FirstHitGBLRefitterDriver.java:253]: Looks like a closing curly brace was added without a counter opening one? I am probably missing something here.

File: analysis/src/main/java/org/hps/analysis/MC/KeepTruthRawTrackerHits.java [hunk 2]
```diff
@@ -80,4 +78,4 @@ protected void process(EventHeader event) {
         }
         event.put(stripHitOutputCollectionName, truthStripHits, SiTrackerHitStrip1D.class, 0);
     }
-}
\ No newline at end of file
+}
```
Comment (cbravo135) [inline analysis/src/main/java/org/hps/analysis/MC/KeepTruthRawTrackerHits.java:81]: Looks like this happens for a bunch of files.

File: analysis/src/main/java/org/hps/analysis/dataquality/MonsterEventAnalysis.java [hunk 3]
```diff
@@ -183,4 +179,4 @@ public void process(EventHeader event){
             }
         }
     }
-}
\ No newline at end of file
+}
```
Comment (cbravo135) [inline analysis/src/main/java/org/hps/analysis/dataquality/MonsterEventAnalysis.java:182]: I really like all this clean up, but don't understand why these new lines with closing braces were added. If you help me understand this I am ready to approve.

**Comment by cbravo135** [comment]
I really think this is probably me not understanding why github is marking it the way it is.

**Comment by cbravo135** [comment]
Ok, talking with my room mate, pretty sure this means you removed the \n at the end of the files that had one. Looks good to me!

**Review by cbravo135 — APPROVED** [review]

**Comment by JeremyMcCormick** [comment]
Okay, I think the newlines ARE supposed to be there. Do we care right now
that my vim ate them? I can fix it up later if needed....

**Pull request merged**